### PR TITLE
Added aliases for spaces 50 and 52

### PR DESCRIPTION
**Pull request by jford1906**
Deleted redundant spaces 172 and 173, properties that were included there are verified in spaces 50 and 52.

### CODE REVIEW

**Review by StevenClontz — COMMENTED** [review]
run `rm .DS_Store` and `rm spaces/.DS_Store`

**Comment by jford1906** [comment]
I've removed those and pushed to my repo

**Review by StevenClontz — APPROVED** [review]

File: spaces/S000052/README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ uid: S000052
 slug: relatively-prime-integer-topology
 name: Relatively prime integer topology
 counterexamples_id: 60
+aliases:
+- Golumb Topology
```
Comment (prabau) [inline spaces/S000052/README.md:7]: That should be "Golomb", right?

**Comment by prabau** [comment]
Hi @jford1906 Not sure if you noticed, but it appears the latest change added a typo in the name of Golomb.

**Comment by jford1906** [comment]
Sorry, missed that. I'll send a new pull

**Pull request merged**